### PR TITLE
DOC-2729 micro app templates

### PR DESCRIPTION
**Pull request by Georgina-gh**
Adding some adaptive card templates with instructions to create micro-apps.

Internal ticket: https://lumapps.atlassian.net/browse/DOC-2729

Any improvements welcome :)

The instructions markdown pages link to json adaptive card files in this repo and to an API documentation page generated with Postman.

Postman collection: [Micro-app API call templates.postman_collection.json](https://github.com/user-attachments/files/16246984/Micro-app.API.call.templates.postman_collection.json)

Public link: https://documenter.getpostman.com/view/32450244/2sA3kPq5Do#intro

### CODE REVIEW

**Comment by Georgina-gh** [comment]
Content reviewed for typos, etc by Beltran from docs team

**Review by bguivarch — APPROVED** [review]
LGTM

File: micro-app/SAP leave request/README.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+    - 1 branch for when the leave request fails
+6. An Adaptive Card for each branch, to display either an error or a success message.
+
+> **Prerequisite:** Create a new micro-app according to [Declare your micro-app extension](https://developer.lumapps.com/portal/documentation/micro-app-steps/declare-your-microapp-extension.md).
+
+Create the elements of the workflow according to the steps in this page and the Developer Portal documentation.
+
+## Step 1: Get the current date
+
+start your workflow with a public API that retrieves the current date. We need the current date in `YYYY-MM-DD` format for the next step.
```
Comment (AndrooCla) [inline micro-app/SAP leave request/README.md:26]:
```suggestion
start your workflow with a public API that retrieves the current date. You need the current date in `YYYY-MM-DD` format for the next step.
```

File: micro-app/ServiceNow ticket creation/README.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+<!-- Documentation to be added to create a first API Call to authenticate when using a connector with the "Basic" option, that will be available in the new connector UI?
+
+Get user ID (required in basic authentication)
+GET https://${snow-instance}.service-now.com/api/now/table/sys_user?sysparm_query=email=${email} to retrieve user id (sys_id) -->
+
+## Step 2: API Call to get the ServiceNow profile information of the user
+
+> **Note:** For more information about ServiceNow APIs, refer to the [Table API](https://docs.servicenow.com/bundle/washingtondc-api-reference/page/integrate/inbound-rest/concept/c_TableAPI.html#d228766e1289) reference.
+
+We can set the reporter of a ticket by identifying the accountId of a user, based on the email used in their LumApps account.
```
Comment (AndrooCla) [inline micro-app/ServiceNow ticket creation/README.md:36]:
```suggestion
You can set the reporter of a ticket by identifying the accountId of a user, based on the email used in their LumApps account.
```

File: micro-app/Workday leave request/README.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+    - 1 branch for when the leave request fails
+6. An Adaptive Card for each branch, to display either an error or a success message.
+
+> **Prerequisite:** Create a new micro-app according to [Declare your micro-app extension](https://developer.lumapps.com/portal/documentation/micro-app-steps/declare-your-microapp-extension.md).
+
+Create the elements of the workflow according to the steps in this page and the Developer Portal documentation.
+
+## Step 1: Get the worker ID of the user
+
+In this example, users have the same value for their LumApps employeeId as their Workday workerId. So we use a LumApps API to retrieve this ID.
```
Comment (AndrooCla) [inline micro-app/Workday leave request/README.md:26]:
```suggestion
In this example, users have the same value for their LumApps employeeId as their Workday workerId. Therefore, you use a LumApps API to retrieve this ID.
```

**Review by AndrooCla — APPROVED** [review]
just minor wording suggestions

**Pull request merged**